src/constants.ts

```diff
@@ -33,7 +33,7 @@ export const ETHER_ADDRESS =
 /**
  * Contract representation of currency types in Juicebox.
  */
-export const JB_CURRENCIES = {
+export const JB_CURRENCIES: { ETH: 1n; USD: 2n } = {
   ETH: 1n,
   USD: 2n,
 };
```

Comment:
@wraeth-eth niiice addition. I think we can do this though to the same effect. WDYT:

```suggestion
export const JB_CURRENCIES = {
  ETH: 1n,
  USD: 2n,
} as const;
```